src/pages/Trace/Review/Review.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useRef, useEffect } from 'react';
+import { NavButton } from 'components/NavButton/NavButton';
+import './Review.css';
+
+export const Review: React.FC = () => {
+    const keepButton: NavButton = {label: 'KEEP', to: '/Trace/Instructions' };
+    // TODO: Delete button should dispatch event that removes the most recent path
+    const deleteButton: NavButton = {label:'DELETE', to: '/Trace/Instructions'};
+
+    const canvasRef = useRef(document.getElementsByClassName('canvasContainer')[0]);
+    useEffect(() => {
+        canvasRef.current = document.getElementsByClassName('canvasContainer')[0];
+        canvasRef.current.classList.add('canvasContainerBackground');
```

Comment:
I find it hard to review the traced lines in the review page, because it is greyed out a little too darkly. Could we no grey it at all, or grey it more lightly, so that we can inspect the lines?

Reply:
I was actually thinking the same thing. I'll remove the grey for now.